command/command_test.go

```diff
@@ -0,0 +1,44 @@
+package command
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+)
+
+func TestExecuteProgram(t *testing.T) {
+
+	cat := ExternalCommand{
+		Path: "cat",
```

Comment:
Please use your own binary that:

When started, prints some stuff on stdout/err but doesn't exit.

Has a signal handler that catches signal sent by test to print some more stuff

Has a signal handler that catches a _different_ signal sent by test and exits.